### ISSUE TITLE
MIMEType is mixed with extension in OutputFileNodeModel

### ISSUE THREAD

**Issue by chahuistle** [genericworkflownodes/GenericKnimeNodes#97]
The OutputFileNodeModel refuses to work when given matching mime types. The problem lies in this line of code, I think:

``` java
final String incomingMimeType = MIMEMap.getMIMEType(((URIPortObjectSpec) inSpecs[0]).getFileExtensions().get(0));
```

Refer to the attached image. It's not possible to get this working even if both Nodes point to files with the same extension. I've already got a fix, but I just want to make sure this is a bug and not some configuration problem on my end.

![screen shot 2013-10-28 at 6 42 32 pm](https://f.cloud.github.com/assets/1537189/1422331/c60c7fe4-3ff9-11e3-86be-6cd4abfb1e75.png)

**Comment by aiche** [comment]
The problem is not the way we query for the mime type but which mime types are registered. In some cases it can happen, that there is a `text/csv` and a `text/CSV` mime type which are incompatible in the current form.

So we obviously need to improve the way we compare the mime types of two files.

**Comment by aiche** [comment]
should be fixed since 5d0397582ce8db0e3159fff98e3de5b75c995023
